nifi-nar-bundles/nifi-registry-bundle/nifi-registry-service/pom.xml

```diff
@@ -52,5 +52,20 @@ language governing permissions and limitations under the License. -->
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.networknt</groupId>
+            <artifactId>json-schema-validator</artifactId>
+            <version>1.0.87</version>
```

Comment:
This should be version `1.1.0`.

Reply:
Corrected. Should this dependency be pulled to a `<dependencyManagement>` section in a parent common both to `nifi-nar-bundles/nifi-standard-bundle/pom.xml` and `nifi-nar-bundles/nifi-registry-bundle/nifi-registry-service/pom.xml`so the version could be specified only once?

Reply:
Thanks for the update. This one could go either way. For now, given just two references, it seems alright to leave it as it stands, but if there are any new modules that use it, it makes sense to promote it to a higher level.